lmdeploy/pytorch/backends/cuda/apply_rotary_emb.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from lmdeploy.pytorch.kernels.cuda import apply_rotary_pos_emb
```

Comment:
Here llama directly use `apply_rotary_pos_emb` in `lmdeploy.pytorch.kernels.cuda` instead of `apply_rotary_pos_emb` in `lmdeploy.pytorch.kernels`. Is it possible that `apply_rotary_pos_emb` in `lmdeploy.pytorch.kernels` will not  be used and `lmdeploy/pytorch/kernels/apply_rotary_pos_emb.py` can be deleted? This is also the issue with other kernels.

Reply:
Yes, that is expected.